src/tools/pbs_python.c

```diff
@@ -2310,11 +2310,37 @@ main(int argc, char *argv[], char *envp[])
 		char *largv[3];
 		int ll;
 		char *pc, *pc2;
```

Comment:
If you are going to go to the trouble of NULLing pc and pc2 (and setting ll=0) below, you should initialize them as well.  It won't matter since you always set them before you use them.